src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
+/* eslint-disable no-console */
 'use strict';
 
-// write your code here
+const logoElement = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
+  logoElement.addEventListener('click', () => {
+    resolve('logo was clicked');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Timeout after 3 seconds'));
+  }, 3000);
+});
+
+function appendMessage(text, className) {
+  const messageDiv = document.createElement('div');
+
+  messageDiv.className = className;
+  messageDiv.textContent = text;
+  document.body.appendChild(messageDiv);
+}
+
+promise1
+  .then((message) => {
+    appendMessage('Promise was resolved!', 'message');
+    console.log(message);
+  })
+  .catch((error) => {
+    appendMessage('Promise was rejected!', 'message error-message');
+    console.error(error);
+  });
+
+promise2
+  .then((message) => {
+    appendMessage('Promise was resolved!', 'message');
+    console.log(message);
+  })
+  .catch((error) => {
+    appendMessage('Promise was rejected!', 'message error-message');
+    console.error(error);
+  });
```

Comment:
Both promises are handled separately, which is fine, but if you want to handle them together, consider using `Promise.race()` or `Promise.allSettled()` depending on your requirements. This is not a mistake, but a suggestion for potential improvement if you need to handle them in a combined manner.